docs/features/kubernetes/authentication.md

```diff
@@ -130,6 +131,36 @@ page for the cluster resource, go to `Overview` > `Properties` tab >
 `Networking` section and copy paste the API server address directly in that
 `url` field.
 
+### Microsoft
+
+The Microsoft provider addresses the topoloy where a custom Microsoft Entra App and [OpenID Connect][9] is used to authenticate to Kubernetes clusters hosted on-premise or in any alternative cloud provider to Azure (e.g. [Amazon EKS][10]);
```

Comment:
```suggestion
The Microsoft provider addresses the topology where a custom Microsoft Entra App and [OpenID Connect][9] is used to authenticate to Kubernetes clusters hosted on-premise or in any alternative cloud provider to Azure (e.g. [Amazon EKS][10]);
```

Can we also update these to be inline links? We don't use this format in our docs generally.

Reply:
I do acknowledge that this convention is being used on this page. I'll log an issue for a follow up to remove this.

Reply:
I do acknowledge that this convention is being used on this page. I'll log an issue for a follow up to remove this.